guides/guides.css

```diff
@@ -132,6 +132,83 @@ html {
     font-weight: 500;
     letter-spacing: 0.02rem;
 }
+.python-img1{
+    height: 350px;
```

Comment:
When you add css in this guides.css file for any doc, add a comment that this block contains css for that particular doc

Reply:
Yeah Sorry I forget it . I will add it.